<throwable>
  <class>org.frankframework.core.SenderException</class>
  <message>exception processing message using request processor [INSTANCE_NAME-TransactionExceptionInDbPipe]: (RequestProcessorException) RequestProcessor [INSTANCE_NAME-TransactionExceptionInDbPipe] caught exception: Pipe [fixedInsert4] unable to execute SQL statement, transaction has been marked as failed by an earlier sender</message>
  <cause>
    <throwable>
      <class>nl.nn.adapterframework.dispatcher.RequestProcessorException</class>
      <message>RequestProcessor [INSTANCE_NAME-TransactionExceptionInDbPipe] caught exception</message>
      <cause>
        <throwable>
          <class>org.frankframework.core.ListenerException</class>
          <message>Pipe [fixedInsert4] unable to execute SQL statement, transaction has been marked as failed by an earlier sender</message>
          <cause>
            <throwable>
              <class>org.frankframework.core.PipeRunException</class>
              <message>Pipe [fixedInsert4] unable to execute SQL statement, transaction has been marked as failed by an earlier sender</message>
            </throwable>
          </cause>
        </throwable>
      </cause>
    </throwable>
  </cause>
</throwable>
